internal/v1/api.yaml

```diff
@@ -588,15 +588,19 @@ components:
       enum: ['aws', 'gcp', 'azure', 'aws.s3']
     AWSUploadRequestOptions:
       type: object
-      required:
-        - share_with_accounts
       properties:
         share_with_accounts:
           type: array
           example: ['123456789012']
           items:
             type: string
           uniqueItems: true
+        share_with_sources:
+          type: array
+          example: ['12345']
+          items:
+            type: string
+          uniqueItems: true
```

Comment:
So none of these 2 options are going to be required from now on. Is that intentional?

Reply:
Yes, but I need to add a check to have at least 1 of them specified.